packages/react-performance/src/performance-mark.ts

```diff
@@ -1,8 +1,6 @@
 import {usePerformanceEffect} from './performance-effect';
 
-export type Stage = 'complete' | 'usable';
-
-export function usePerformanceMark(stage: Stage, id: string) {
+export function usePerformanceMark(stage: string, id: string) {
```

Comment:
why remove the type? I thought it was pretty useful.

Reply:
The API actually supports any arbitrary string. I guess we could export the magic ones as an enum though

Reply:
The API actually supports any arbitrary string. I guess we could export the magic ones as an enum though

Reply:
Added back as an enum, still using `string` as the type here so it properly supports any custom stage value